src/dropdown/styles.ts

```diff
@@ -21,9 +21,17 @@ type DropdownStyleOptions = {
   x: number | null;
   y: number | null;
   inverted: boolean;
+  limitHeight: boolean;
```

Comment:
```suggestion
  shouldLimitHeight: boolean;
```
makes it more clear that the value is a boolean, rather than a pixel amount